### DISCUSSION TITLE
Simplifying sums of min terms

### DISCUSSION THREAD

**Discussion by alexreinking** [sympy/sympy#22601] [Q&A]
I'm new to SymPy, so this might actually be easy, but I couldn't find anything in the documentation that quite matched. I'm trying to simplify the following expression:

```
>>> from sympy import *
>>> i, N = symbols('i N', integer=True, negative=False)
>>> summation(Min(N-8*i, 8), (i, 0, ceiling(N/8)-1)).simplify()
Sum(Min(8, N - 8*i), (i, 0, ceiling(N/8) - 1))
```

This could simplify to just `N` (and is true in general for a positive `k`, in place of `8`). Is there some way I can get this to happen?

**Comment by oscargus** [comment]
I do not think that there is any support for that in SymPy at the moment. In general, the support for variables taking on a given range (except positive/negative) is quite limited in SymPy. I assume that one here would like to do something like:

`refine(Min(8, N - 8*i), 0 <= i <= ceiling(N/8) - 1)` to simplify the summation term and that is currently not supported. First, as there is no refine-functionality for Min/Max, second, as there isn't a way to express `i <= ceiling(N/8) - 1` (i integer and non-negative is supported though, but doesn't help).
